bad.php

```diff
@@ -0,0 +1,6 @@
+<?php
+class SomeClass {
+    public function bad_braces (  ) {
```

Comment:
Method name "SomeClass::bad_braces" is not in camel caps format
Expected "function abc(...)"; found "function abc (...)"
Expected 0 spaces between brackets of function declaration; 2 found
Expected 0 spaces before opening parenthesis; 1 found
Opening brace should be on a new line